cves/CVE-2016-2327.yml

```diff
@@ -192,8 +202,11 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    This vulnerability can be exploited with a custom .avi file that would result in the last_frame object having no
+    items in the linesize array, which seems reasonable for a thorough automated tester to be able to discover, as long
+    as the tool is testing .avi files.
```

Comment:
This is good - actually some of these details would be nice to incorporate into the opening description. I think your audience could use more explanation on what the code is doing. Your audience isn't looking at the code and you're distilling your knowledge for them. And "testing avi files" is pretty broad, could use some more specificity.